internal/api/shared/schemas/schemas.go

```diff
@@ -80,22 +75,26 @@ func New(conf config.Schemas) Schemas {
 			baseDef:     basePanelDefVal,
 			schemas:     &sync.Map{},
 			schemasPath: conf.PanelsPath,
-			kindCuePath: fmt.Sprintf("%s.%s", panelDefPath, kindField),
+			kindCuePath: "#panel.spec.plugin.kind",
 		},
-		queries: &cueDefs{
-			context:     ctx,
-			baseDef:     baseQueryDefVal,
-			schemas:     &sync.Map{},
-			schemasPath: conf.QueriesPath,
-			kindCuePath: fmt.Sprintf("%s.%s", datasourceDefPath, kindField),
+		queries: &cueDefsBis{
+			cueDefs: cueDefs{
+				context:     ctx,
+				baseDef:     baseQueryDefVal,
+				schemas:     &sync.Map{},
+				schemasPath: conf.QueriesPath,
+				kindCuePath: "spec.plugin.kind",
```

Comment:
why not `#spec.plugin.kind` ?

Reply:
before you could you either use one or the other, but with the changes there's no longer `#spec` (in our case I found it was more levant to use the `close(...)` clause instead.

Reply:
ah interesting !